recipes/seqwish/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "0.2.1" %}
+
+package:
+  name: seqwish
+  version: '{{version}}'
+
+source:
+  git_url: https://github.com/ekg/seqwish
```

Comment:
Please prefer the release from `https://github.com/ekg/seqwish/archive/v0.2.1.tar.gz` over the direct Git checkout. Then you also specify a checksum.

Reply:
I tried that in some of my earlier edits. There seems to be a large number of integrated dependencies :(

Reply:
Oh ok, it appears that the problem is that [GitHub releases do not include Git submodules](https://github.community/t5/How-to-use-Git-and-GitHub/Create-Release-that-contains-submodule/td-p/12073). Then I’m fine with this.